extensions/liteproto/pom.xml

```diff
@@ -96,6 +96,31 @@
             </path>
           </annotationProcessorPaths>
         </configuration>
+        <executions>
```

Comment:
It looks like the core Truth build is the only one with `useIncrementalCompilation` or `moduleVersion`. Once we're settled on what to use there, is it worth using for the others? (It's probably possible to share the configuration with `dependencyManagement`, but I don't really have an opinion either way on that.)